fieldbehavior/fieldbehavior_test.go

```diff
@@ -7,14 +7,28 @@ import (
 	"google.golang.org/genproto/googleapis/api/annotations"
 	"google.golang.org/genproto/googleapis/example/library/v1"
 	"google.golang.org/protobuf/types/known/fieldmaskpb"
+	"google.golang.org/protobuf/types/known/timestamppb"
 	"gotest.tools/v3/assert"
 	"gotest.tools/v3/assert/cmp"
 )
 
-func TestValidateRequiredFields(t *testing.T) {
+func TestClearFields(t *testing.T) {
 	t.Parallel()
-	assert.NilError(t, ValidateRequiredFields(&examplefreightv1.GetShipmentRequest{Name: "testbook"}))
-	assert.Error(t, ValidateRequiredFields(&examplefreightv1.GetShipmentRequest{}), "missing required field: name")
+	t.Run("clear fields with set field_behavior", func(t *testing.T) {
+		t.Parallel()
+
+		site := &examplefreightv1.Site{
+			Name:        "site1",           // has no field_behaviors; should not be cleared.
+			CreateTime:  timestamppb.Now(), // has OUTPUT_ONLY field_behavior; should be cleared.
+			DisplayName: "site one",        // has REQUIRED field_behavior; should not be cleared.
+		}
+
```

Comment:
IMO one t.Run should be readable as one paragraph, i.e. blank lines between t.Runs but now within